clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala

```diff
@@ -142,10 +142,16 @@ class MoveExecutor[CI <: ClioIndex](protected val moveCommand: MoveCommand[CI])(
     * more IO operations.
     */
   protected[dispatch] def buildMove(
-    metadata: moveCommand.index.MetadataType
+    metadata: moveCommand.index.MetadataType,
+    deliverMetrics: Boolean = false
```

Comment:
According to DSDEGP-2229 the sample level metrics feature is for deliver only, not move.

Reply:
Because of the way clio-client is structured, the deliver stuff uses the move trait, which means we have to pass this option to all move implementers :(